pages/_app.js

```diff
@@ -1,8 +1,12 @@
 import GlobalStyle from "../styles";
+import Head from "next/head";
 
 export default function App({ Component, pageProps }) {
   return (
     <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Actually I am not quite sure but I thiiiiiink, you do not need this. Do not quote me on that :D

Reply:
I'm also not sure if I need it or not 🤷‍♀️ 😄

Reply:
Maybe you are right, I just removed that part of the code and the display of the UI is better now :)